iocore/net/UnixNetVConnection.cc

```diff
@@ -669,6 +638,9 @@ UnixNetVConnection::do_io_write(Continuation *c, int64_t nbytes, IOBufferReader
 void
 UnixNetVConnection::do_io_close(int alerrno /* = -1 */)
 {
+  // FIXME: the nh must not nullptr.
+  ink_assert(!nh);
```

Comment:
This assertion is failed.
```
Fatal: UnixNetVConnection.cc:642: failed assertion `!nh`
```
https://ci.trafficserver.apache.org/job/linux-github/3931/console

Reply:
It is a typo mistake. updated, please review. thanks.